docs/guide/replacement-guides/faker.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Modern  replacements for the unmaintained faker package generating massive amounts of fake (but realistic) data
+---
+
+# Replacements for `faker`
+
+## `@faker-js/faker`
+
+[`@faker-js/faker`](https://github.com/faker-js/faker) is a direct, community‑maintained fork of `faker` with new features, bugfixes, modern ESM/CJS builds, and updated data/locales.
+
+```ts
+const faker = require('faker') // [!code --]
+const { faker } = require('@faker-js/faker') // [!code ++]
+
+// Name → Person
+const name = faker.name.findName() // [!code --]
+const name = faker.person.fullName() // [!code ++]
+
+// “createCard” → compose your own object
+const card = faker.helpers.createCard() // [!code --]
+const user = { // [!code ++]
```

Comment:
maybe the example should use something that existed in both?

it looks like they deprecated `createCard` but thats unrelated to the transition from the old lib to the fork (i.e. it happened later as a progression of the API)